.changeset/chatty-months-report.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/techdocs-common': minor
```

Comment:
While we're on the `0.x.y` series, we're treating minor API additions as patch and breaking changes as minor.  So I'd recommend setting these both to `patch` for now.